events.ics

```diff
@@ -8,7 +8,8 @@ METHOD:PUBLISH
 {% for event in sorted %}BEGIN:VEVENT
 UID:{{ event.url | prepend: site.baseurl | prepend: site.url }}
 DTSTART;TZID={{ site.timezone }}{% if event.time_tbc %};VALUE=DATE:{{ event.date | date: "%Y%m%d" }}{% else %}:{{ event.date | date: "%Y%m%dT%H%M%S" }}{% endif %}
-DTSTAMP{% if event.time_tbc %};VALUE=DATE:{{ event.date | date: "%Y%m%d" }}{% else %}:{{ event.date | date: "%Y%m%dT%H%M%S" }}{% endif %}
+{% if event.end_date %}DTEND;TZID={{ site.timezone }}{% if event.time_tbc %};VALUE=DATE:{{ event.end_date | date: "%Y%m%d" }}{% else %}:{{ event.end_date | date: "%Y%m%dT%H%M%S" }}{% endif %}{% endif %}
+DTSTAMP:{{ event.date | date: "%Y%m%dT%H%M%S" }}
```

Comment:
This appears to lose some of the support for `time_tbc` -- could we validate that that still works?

Reply:
`DTSTAMP` is the creation time for the event, so it doesn't really need the concept of `time_tbc`. ea3c018a179b3c17ba4956cd91ec1ae6949270a9